### PR TITLE
minor edits + PvD to the main document

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

File: draft-brw-scone-rate-policy-discovery.md [hunk 1]
```diff
@@ -690,6 +653,33 @@ To discover a network rate-limit policy, the DHCP client includes OPTION_V4_NRLP
...
+         "cir":50,
+         "cbs":10000,
+         "ebs":20000
+      }
+   ]
+}
+~~~~~
+{: #pvd-ex title="NRLP Example with PvD"}
+
+PvD NRLP object can be extended by defining new attributes (e.g., supply an ADN and locators of a network entity to negotiate advanced features with the network, reachability information of a network API to invoke differentiated forwarding behaviors, a slice identifier {{TS-23.501}}{{?RFC9543}}, etc.).
```
Comment (tireddy2) [inline draft-brw-scone-rate-policy-discovery.md:685]: what do you mean by "ADN" ?

**Review by tireddy2 — COMMENTED** [review]
Most of the changes look good.

**Review by tireddy2 — COMMENTED** [review]
I think security considerations section also needs to be updated to discuss about PvD.

File: draft-brw-scone-rate-policy-discovery.md (hunk 1 above)
Comment (boucadair) [reply draft-brw-scone-rate-policy-discovery.md:685]: This is to focus on the extensibility feature of PvD.

PvD can provide the required information to locate a network API and thus negotiate more "metadata"-related operation, especially for the host to network direction. That direction is out of scope of scope; we may remove this if it is confusing.

Comment (tireddy2) [reply draft-brw-scone-rate-policy-discovery.md]: Please remove it.

Comment (boucadair) [reply draft-brw-scone-rate-policy-discovery.md]: Deal!

Shouldn't we highlight that PvD object can be easily extended?

**Pull request merged**